packages/web/src/scss/components/TextArea/README.md

```diff
@@ -71,9 +71,28 @@ modifier to fill the available space.
 ## Validation States
 
 Validation states can be presented either by adding a CSS modifier class
-(`TextArea--success`, `TextArea--warning`, `TextArea--error`), or by adding
+(`TextArea--success`, `TextArea--warning`, `TextArea--danger`, `TextArea--error`), or by adding
```

Comment:
shouldn't we mention the deprecation here?

Reply:
done